README.md

```diff
@@ -1,2 +1,92 @@
+
+## INTRODUCTION
+The carpics spinner is a 360 viewer which allows cycling through a reel of photos, and selectively zooming in for emphasis.  The cycling can either be automatic, or manually controlled using mouse actions.  It can also be configured to have automatic spin with manual controls as well ( manual controls take precedence in this case).  Mobile events are fully implemented as well.  
+
+### Desktop Controls:
+
+Mouse click and drag to rotate.
+Mouse doubleclick (mouse1 only) to zoom/unzoom.
+Hold mouseclick without movement to zoom.
+
+### Mobile Controls:
+
+Touch and drag to rotate.
+Touch and hold to zoom/unzoom.
+
+## Further configurations are :
+
+- The number of web connections that are usable for this utility (more connections speed up loading via concurrence, at the expense of higher peak bandwidth requirements) - also prevents saturation of connections, which decreases page performance.  
+- Whether it should spin before all images are loaded, if autospinning. 
+- If mouse interaction is not desired at all – can disable mouse input. 
+- Configurable mouse sensitivity, time between autospin frames. 
+
+## Other features: 
+- Endpoint agnostic – gives a JS API to make your own spinners!  Only requires configuration object and array of image URLS. 
+- Fault tolerant – Broken images should not display, nor should they break the spinner. 
+ 
+## How to use: 
+Include JQuery, as this library is Jquery dependent (for now - may revisit this in later versions).  Insert Jquery as such:
+Desktop Jquery:
+```
+	<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">  
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+```
+Mobile Jquery:
+```
+    <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jquerymobile/1.4.5/jquery.mobile.min.css">  
+    <script src="https://ajax.googleapis.com/ajax/libs/jquerymobile/1.4.5/jquery.mobile.min.js"></script>
+```
+**\*\*Note!\*\*** :  Jquery Mobile's CSS sheet makes changes to the way Jquery events are handled.   If the jquery css is new to your site, test.  The CSS can be omitted without causing issues for this library if issues are caused by including it.  This is especially important if you use a responsive design - as most issues caused are most apparent in desktop.
+
+After including Jquery, the basic way to use this library is to provide a div with a few mandatory properties. Here is an example of the minimal spinner div:
+```
+<div id="<UniqueDivId>" CarPicsUrl="<image array resource>" class="CarPicsSpinner" >
+</div>
+```
+
+In the same HTML document as this is defined, then add as well a script tag referencing to the spinner source:
+```
+<script src="<Host Path>/CarPicsSpinner.js" async=true></script>
+```
+### Mandatory Attributes:
+- id : value=<string>  - Div needs ID to be uniquely identified by the library.  Must be unique, but is not limited besides that. 

```

Comment:
Yes.  It is just ID - any string is valid.  It is HTML ID attribute, so as long as the user doesn't violate HTML standards (multiple divs with the same ID) it will work fine.